auth-web/src/routes/index.ts

```diff
@@ -99,9 +99,15 @@ router.beforeEach((to, from, next) => {
         case LoginSource.BCEID:
           // eslint-disable-next-line no-console
           console.log('[Navigation Guard] Redirecting user to TOS since user has not accepted one')
+          // if there's redirectUri in query string, keep existing redirectUri, otherwise use current location
+          const urlParams = new URLSearchParams(window.location.search)
+          var uriRedirectTo = urlParams.get('redirectUri')
```

Comment:
var does the job but we have used 'let' extensively in sbc-auth for re-assigning variables

Reply:
updated.